conf/modules.config

```
@@ -1200,7 +1203,7 @@ process {
     }
 
     withName: SAMTOOLS_INDEX_DAMAGE_RESCALED {
-        tag = { "${meta.reference}|${meta.sample_id}_${meta.library_id}" }
+        tag = { "${meta.reference}|${meta.damage_manipulation}|${meta.sample_id}_${meta.library_id}" }
```

Keep attribute order in tag consistent. 
`reference|sample_id|dmg_manip`, like in `MERGE_LIBRARIES_DAMAGE_MANIPULATION:SAMTOOLS_MERGE_LIBRARIES`

```suggestion
        tag = { "${meta.reference}|${meta.sample_id}_${meta.library_id}|${meta.damage_manipulation}" }
```